python/ray/air/tests/test_tensor_extension.py

```diff
@@ -676,6 +779,27 @@ def test_variable_shaped_tensor_array_uniform_dim():
         np.testing.assert_array_equal(a, expected)
 
 
+@pytest.mark.parametrize("tensor_format", ["v1", "v2"])
+def test_large_arrow_tensor_array(restore_data_context, tensor_format):
+    DataContext.get_current().use_arrow_tensor_v2 = tensor_format == "v2"
+
+    test_arr = np.ones((1000, 550), dtype=np.uint8)
+
+    if tensor_format == "v1":
+        with pytest.raises(ArrowConversionError) as exc_info:
+            ta = ArrowTensorArray.from_numpy([test_arr] * 4000)
+
+        assert (
+            repr(exc_info.value.__cause__)
+            == "ArrowInvalid('Negative offsets in list array')"
+        )
+    else:
+        ta = ArrowTensorArray.from_numpy([test_arr] * 4000)
+        assert len(ta) == 4000
+        for arr in ta:
+            assert np.asarray(arr).shape == (1000, 550)
+
```

Comment:
Nit: IMO this would be more readable as two separate tests because separate tests would avoid conditionals, and there's only one shared line between the two parameterizations

Reply:
I definitely see your point, however splitting it in 2 tests obstructs the intention of it which is to show that it doesn't work w/ V1 and does so w/ V2